projects/sound-open-firmware/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+#
+# Defines a docker image that can build sound open firmware
+#
+# Usage:
+# check out sof
+# build docker image:
+# docker build --build-arg UID=$(id -u) -t sof .
+# docker run -it  -v <insert sof dir here>:/home/sof/workdir --user `id -u` sof
+#
+# For incremental builds:
+# docker run -it  -v <insert sof dir here>:/home/sof/work/sof.git --user `id -u` sof ./incremental.sh
+#
+
+FROM gcr.io/oss-fuzz-base/base-builder
+
+RUN apt-get -y update && \
+	    apt-get install -y \
+	    	autoconf \
+		bison \
+		build-essential \
+		flex \
+		gawk \
+		gettext \
+		git \
+		gperf \
+		help2man \
+		libncurses5-dev \
+		libssl-dev \
+		libtool \
+		libtool-bin \
+		pkg-config \
+		software-properties-common \
+		texinfo \
+		udev \
+		cmake \
+		libglib2.0-dev
+
+ARG CLONE_DEFAULTS="--depth 5"
+
+# Use ToT alsa utils for the latest topology patches.
+RUN cd /tmp && \
```

Comment:
this should be in build.sh so that they are built with the right sanitizer instrumentation.

Reply:
This was based of another chromeos project (cras) which does something similar https://github.com/google/oss-fuzz/blob/master/projects/cras/Dockerfile#L45
Also I believe these are not actually compiled in and are just used for the tooling.

Reply:
ok fine then.